src/Facebook/InstantArticles/Parser/instant-articles-rules.json

```diff
@@ -271,6 +271,15 @@
                 }
             },
             {
+                "class": "AuthorRule",
```

Comment:
If you put this rule above the other, it wont be necessary to use the `not(a)` into the selector.